### PR TITLE
Add missing empty p2sh input script to svp spend transaction

### PR DESCRIPTION
**Pull request by julia-zack**
The empty input script has to be added for every svp spend tx input.

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinUtilsTest.java [hunk 1]
```diff
@@ -440,6 +438,26 @@ void addInputFromOutputSentToScript_withMatchingOutputScript_shouldAddInputWithO
...
     }
 
+    @Test
+    void addP2SHEmptyInputScript_shouldAddExpectedScriptSigToInput() {
+        // arrange
+        Federation federation = P2shErpFederationBuilder.builder().build();
+        Script redeemScript = federation.getRedeemScript();
+
+        BtcTransaction transaction = new BtcTransaction(btcMainnetParams);
+        transaction.addInput(transaction.getHash(), 0, redeemScript);
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinUtilsTest.java:448]: This line raises a few questions
- Is that method expecting a redeem script or a script sig? I'd debug this and check the transaction after adding the input
- If it's expecting a redeemScript, does it use it to build an empty scriptsig? Does this work with ERP type scripts? i.e. adding the flo op code
- Is this how we are creating the spend tx? If that is the case then no need to add the empty script sig, right?

File: rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java [hunk 2]
```diff
@@ -1081,11 +1080,18 @@ private BtcTransaction createSvpSpendTransaction(BtcTransaction svpFundTxSigned,
     }
 
     private void addSvpSpendTransactionInputs(BtcTransaction svpSpendTransaction, BtcTransaction svpFundTxSigned, Federation proposedFederation) {
+        Script proposedFederationRedeemScript = proposedFederation.getRedeemScript();
         Script proposedFederationOutputScript = proposedFederation.getP2SHScript();
         addInputFromMatchingOutputScript(svpSpendTransaction, svpFundTxSigned, proposedFederationOutputScript);
-
-        Script flyoverProposedFederationOutputScript = getFlyoverScriptPubKey(bridgeConstants.getProposedFederationFlyoverPrefix(), proposedFederation.getRedeemScript());
-        addInputFromMatchingOutputScript(svpSpendTransaction, svpFundTxSigned, flyoverProposedFederationOutputScript);
+        svpSpendTransaction.getInput(0)
```
Comment (marcos-iov) [inline rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java:1086]: Should we also update the tests where the spend tx is created? Make sure it has the expected script sig after creation

Comment (julia-zack) [reply rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java:1086]: i think thats covered with the test that tests the method, but there's no harm in adding it sure

Comment (julia-zack) [reply rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java:1086]: updated

**Pull request merged**